File: Anyelo Carrillo php/diccionario.php
<?php
// Diccionario en PHP (se usa un array asociativo para almacenar datos del coche)
$coche = array(
    "marca" => "Ford",
    "color" => "rojo",
    "modelo" => "sedan",
    "placa" => "ROS227"
);

// Imprimir el color actual del coche
echo $coche["color"] . "\n";

// Cambiar el valor del color
$coche["color"] = "verde";
echo $coche["color"] . "\n"; // Ahora el color es verde

// Imprimir la marca actual del coche
echo $coche["marca"] . "\n";

// Cambiar el valor de la marca
$coche["marca"] = "Renault";
echo $coche["marca"] . "\n"; // Ahora la marca es Renault

// Mostrar el diccionario completo con los cambios realizados
print_r($coche);

File: Anyelo Carrillo Java/Diccionario.java
import java.util.HashMap;
import java.util.Map;

public class Diccionario {
    public static void main(String[] args) {
        Map<String, String> coche = new HashMap<>();
        coche.put("marca", "Ford");
        coche.put("color", "rojo");
        coche.put("modelo", "sedan");
        coche.put("placa", "ROS227");

        System.out.println(coche.get("color"));
        coche.put("color", "verde");
        System.out.println(coche.get("color"));
        System.out.println(coche.get("marca"));
        coche.put("marca", "Renault");
        System.out.println(coche.get("marca"));
        System.out.println(coche);
    }
}
